instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpClientInstrumenterBuilder.java

```diff
@@ -245,7 +246,8 @@ public Instrumenter<REQUEST, RESPONSE> build() {
                 statusExtractorTransformer.apply(HttpSpanStatusExtractor.create(attributesGetter)))
             .addAttributesExtractor(httpAttributesExtractorBuilder.build())
             .addAttributesExtractors(additionalExtractors)
-            .addOperationMetrics(HttpClientMetrics.get());
+            .addOperationMetrics(HttpClientMetrics.get())
+            .setSchemaUrl(SchemaUrls.V1_37_0);
```

Comment:
does setting schema url pose any additional restrictions to what we can do with these instrumentations?

Reply:
we can still emit additional attributes if that what you're asking?